torch/_refs/__init__.py

```diff
@@ -1879,6 +1880,28 @@ def sum(
     )
 
 
+def sum_to_size(
+    a: Tensor,
+    *shape,
```

Comment:
What's the correct type annotation here?

Reply:
We haven't been type annotating the `*shape` so you needn't bother